src/components/Person/Person.jsx

```diff
@@ -1 +1,38 @@
-// export const Person = ({ person }) => ();
+const unmarriedInfo = 'I am not married';
+const maleRelationshipStatus = 'husband';
+const femaleRelationshipStatus = 'wife';
```

Comment:
Static constants are usually written in uppercase:
```suggestion
const UNMARRIED_TEXT == 'I am not married';
const MALE_RELATIONSHIP_STATUS = 'husband';
const FEMALE_RELATIONSHIP_STATUS = 'wife';
```